build.sbt

```diff
@@ -2,6 +2,11 @@ import Dependencies._
 import Path._
 import com.typesafe.tools.mima.core._, ProblemFilters._
 
+val _ = {
+  sys.props += ("line.separator" -> "\n")
+  sys.props += ("file.separaror" -> "/")
```

Comment:
wait, why are we doing this?

Reply:
contraband uses system dependent separator and this means i get tons of diffs in generated files

Reply:
I see - that's not very nice! at the very minimum I'd say we should report this to contraband.

I'm not sure if we should commit this here, but if we do it should link to the contraband issue.